storage/innobase/buf/checksum.cc

```diff
@@ -746,25 +752,18 @@ std::ostream &operator<<(std::ostream &out, const page_id_t &page_id) {
   return (out);
 }
 
-uint32_t
-buf_calc_page_crc32_encrypted_with_keyring(
-	const byte*	page,
-	uint            page_size,
-	bool		use_legacy_big_endian /* = false */)
-{
-	ut_crc32_func_t	crc32_func = use_legacy_big_endian
-		? ut_crc32_legacy_big_endian
-		: ut_crc32;
-
-	const uint32_t	c1 = crc32_func(
-		page + FIL_PAGE_OFFSET,
-		FIL_PAGE_FILE_FLUSH_LSN - FIL_PAGE_OFFSET);
+uint32_t buf_calc_page_crc32_encrypted_with_keyring(
+    const byte *page, uint page_size,
+    bool use_legacy_big_endian /* = false */) {
+  ut_crc32_func_t crc32_func =
```

Comment:
Can this legacy business be dropped? Not in this PR, but please make a note

Reply:
Created PS-5705: Get rid of legacy_big_endian_checksum